doc/GSG/next_steps.rst

```diff
@@ -27,101 +27,152 @@ After installing |short_name|, set the environment variables:
+              #include <oneapi/tbb.h>
             
-            int main (){
-                int sum = oneapi::tbb::parallel_reduce(
-                    oneapi::tbb::blocked_range<int>(1,101), 0,
-                    [](oneapi::tbb::blocked_range<int> const& r, int init) -> int {
-                        for (int v = r.begin(); v != r.end(); v++) {
-                            init += v;
-                        }
-                        return init;
-                    },
-                    [](int lhs, int rhs) -> int {
-                        return lhs + rhs;
-                    }
-                );
+              int main (){
+                  int sum = oneapi::tbb::parallel_reduce(
+                      oneapi::tbb::blocked_range<int>(1,101), 0,
+                      [](oneapi::tbb::blocked_range<int> const& r, int init) -> int {
+                          for (int v = r.begin(); v != r.end(); v++) {
+                              init += v;
+                          }
+                          return init;
+                      },
+                      [](int lhs, int rhs) -> int {
+                          return lhs + rhs;
+                      }
+                  );
       
-                printf("Sum: %d\n", sum);
-                return 0;
-            }
+                  printf("Sum: %d\n", sum);
+                  return 0;
+              }
       
-      #. Open the ``tasks.json`` file in the ``.vscode`` directory and paste the following lines to the args array:
+        #. Open the ``tasks.json`` file in the ``.vscode`` directory and paste the following lines to the args array:
 
-         * ``-Ipath/to/oneTBB/include`` to add oneTBB include directory. 
-         * ``path/to/oneTBB/`` to add oneTBB. 
+           * ``-Ipath/to/oneTBB/include`` to add oneTBB include directory. 
+           * ``path/to/oneTBB/`` to add oneTBB. 
 
-         For example:
+           For example:
 
-         .. code-block::
+           .. code-block::
 
-                { 
-                   "tasks": [
-                        {
-                           "label": "build & run",
-                           "type": "cppbuild",
-                           "group": {
-                           "args": [ 
-                               "/IC:\\Program Files (x86)\\Intel\\oneAPI\\tbb\\2021.9.0\\include",
-                               "C:\\Program Files (x86)\\Intel\\oneAPI\\tbb\\2021.9.0\\lib\\ia32\\vc14\\tbb12.lib"
+                  { 
+                     "tasks": [
+                          {
+                             "label": "build & run",
+                             "type": "cppbuild",
+                             "group": {
+                             "args": [ 
+                                 "/IC:\\Program Files (x86)\\Intel\\oneAPI\\tbb\\2021.9.0\\include",
+                                 "C:\\Program Files (x86)\\Intel\\oneAPI\\tbb\\2021.9.0\\lib\\ia32\\vc14\\tbb12.lib"
```

Comment:
Let's adjust the path to the current layout. 
It should be "C:\\Program Files (x86)\\Intel\\oneAPI\\tbb\\2021.14.0\\lib\\tbb12.lib"

Reply:
Sorry, my bad. There is no 2021.14.0 at all. It's better to have the latest 2022.1.0.